sphinx/testing/util.py

```diff
@@ -91,8 +91,8 @@ def __init__(
         status: IO | None = None,
         warning: IO | None = None,
         tags: list[str] | None = None,
-        docutils_conf: str | None = None,
-        parallel: int = 0,
+        docutilsconf: str | None = None,
+        parallel: int = 4,
```

Comment:
```suggestion
        parallel: int = 0,
```

This default value is also public so don't change it.

Reply:
Reverted this